Omulosi/iReporter
# #162341923 Create unit tests for all the API endpoints

#### What does this PR do?
Create unit tests for all the API endpoints

#### Description of Task to be completed?
Create unit tests for the following API endpoints in the `tests` folder
* GET /api/v1/red-flags
* GET /api/v1/red-flags/<id>
* POST /api/v1/red-flags
* PATCH /api/v1/red-flags/<id>/location
* PATCH /api/v1/red-flags/<id>/location
* DELETE /api/v1/red-flags/<id>

#### How should this be manually tested?
clone the repo, move to the ft-api-endpoints-162341923 branch and run the following command:
```
pytest app/tests/v1/test_views.py
```
check that all the tests pass

#### What are the relevant pivotal tracker stories?
#162341923
